llama-index-integrations/vector_stores/llama-index-vector-stores-milvus/llama_index/vector_stores/milvus/base.py

```diff
@@ -133,6 +134,7 @@ def __init__(
         consistency_level: str = "Strong",
         overwrite: bool = False,
         text_key: Optional[str] = None,
+        metadata_keys: List[str] = [],
```

Comment:
kwargs shouldn't default to a list, it can cause some weird interactions in python. We can declare it as `None` and then set it as an empty list below if it is still None

Reply:
Got it. Thanks :)